scopes/defender/jest/jest.worker.ts

```diff
@@ -1,6 +1,7 @@
 import { stringify, parse } from 'flatted';
 import { expose } from '@teambit/worker';
-import { runCLI } from 'jest';
+
+const jest = require('jest');
```

Comment:
Is it possible to change it to `import jest from 'jest'` ? with `require` syntax we loose the lazy-loading of the babel plugin.